datareduction/datareducer.cpp

```diff
@@ -3138,6 +3138,12 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
          outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_connection",CellParams::CONNECTION,1));
          continue;
       }
+      if(lowercase == "vg_connection_coordinates") {
+         FieldTracing::fieldTracingParameters.doTraceFullBox = true;
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_connection_coordinates_pos",CellParams::CONNECTION_POS_X,3));
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_connection_coordinates_neg",CellParams::CONNECTION_NEG_X,3));
+         continue;
+      }
```

Comment:
I would just include this automatically when vg_connection is active, no need for yet another cfg line.